sustainability/src/main/java/edu/asu/diging/sustainability/web/search/ResearcherSearchController.java

```
@@ -37,7 +60,19 @@ public String search(Model model, @RequestParam("selectedConcepts") String[] sel
             }
         }
         model.addAttribute("concepts", concepts);
-        
+
         return "perspective/researcher/search";
     }
+
+    @RequestMapping(value = "/perspective/researcher/resource", method = RequestMethod.GET)
+    @ResponseBody
+    public IResource reload(@RequestParam("uri") String uri) {
+        IResource resource = null;
+        try {
+            resource = resourceManager.getResource(uri);
+        } catch (NotAValidResourceException e) {
+            logger.error("Resource " + uri + " is not a valid one.");
```

don't swallow the exception!

---

not addressed